ctpolicy/applepolicy.go

```diff
@@ -41,6 +42,25 @@ func (appleP AppleCTPolicy) LogsByGroup(cert *x509.Certificate, approved *loglis
 	return groups, err
 }
 
+// LogsByGroup2 describes submission requirements for embedded SCTs according to
+// https://support.apple.com/en-us/HT205280. Returns data even when error emitted.
```

Comment:
Returning (partial) data + error is unusual, so it might be worth adding to the method comment and explaining why it does this, and which parts of the data can be used/trusted when an error is also emitted.

Reply:
Added explanation comment.

Reply:
Just to capture the IRL chat: if there's not currently a definite use case for having the incomplete set of logs returned, then it's probably safer to just return `nil, err` to avoid a class of potential bugs. 

No problem to pop a `TODO` in and then follow up in another CL if that change would cause too much knock-on change in this one.